### PR TITLE
adding sql firewall rule facts

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Adding SQL Firewall Rule facts module

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME

azure_rm_sqlfirewallrule_facts

##### ANSIBLE VERSION
2.7

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sqlfirewallrule_facts
+version_added: "2.8"
+short_description: Get SQL Firewall Rule facts.
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py]: Azure

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+description:
+    - Get facts of SQL Firewall Rule.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py]: i think the "You can obtain...." sentence is not necessary, but it's minor

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+        required: True
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py]: resoruce group requried?

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py:33]: not support tag?

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py (hunk 3 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py]: yes

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py (hunk 4 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py:33]: no, firewall rule is subresource that doesn't have own tags

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py (hunk 1 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py]: fixed

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py (hunk 2 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py]: removed

**Review by yungezz — APPROVED** [review]

**Pull request merged**